src/components/editMeasure/details/MeasureDefinitions/MeasureDefinitions.tsx

```diff
@@ -249,8 +249,9 @@ const MeasureDefinitions = (props) => {
             <tbody data-testId="measure-definitions-table-body">
               {visibleDefinitions?.length > 0 ? (
                 visibleDefinitions.map((measureDefinition, index) => (
-                  <MeasureDefinitionRow
-                    measureDefinition={measureDefinition}
+                  <MeasureMetaDataRow
+                    value={measureDefinition}
+                    type="definition"
```

Comment:
only if it makes sense please else ignore.
```suggestion
                    name={measureDefinition.term}
                    description={measureDefinition.definition}
```